### PR TITLE
[Merged by Bors] - docs: fix typos across repository

### PR DESCRIPTION
**Pull request by tristan-f-r**
Supercedes https://github.com/leanprover-community/mathlib4/pull/20236. Hopefully this stops a few typos pull requests later.

Uses `crates-ci/typos`, with the following config:

```toml
[default.extend-words]
# keywords & abbreviations
pn = "pn"
hae = "hae"
hom = "hom"
mor = "mor"
thm = "thm"
tpos = "tpos"
hpe = "hpe"
fle = "fle"
eis = "eis"
iy = "iy"
hge = "hge"
alls = "alls"
struc = "struc"
onl = "onl"
hav = "hav"
hax = "hax"
ba = "ba"
ein = "ein"
adn = "adn"
loopup = "loopup"
hsa = "hsa"
dum = "dum"
mapp = "mapp"
nd = "nd"
ist = "ist"
haa = "haa"
ot = "ot"
comon = "comon"
ihs = "ihs"
thr = "thr"
projectives = "projectives"
hda = "hda"
ane = "ane"
hsi = "hsi"
minimals = "minimals"
ninj = "ninj"
ue = "ue"
tne = "tne"
lits = "lits"
ine = "ine"
tre = "tre"
fo = "fo"
nam = "nam"
larg = "larg"
reord = "reord"
nin = "nin"
toi = "toi"
ands = "ands"
hte = "hte"
strat = "strat"
hink = "hink"
lastr = "lastr"
wth = "wth"
iin = "iin"

# deprecations based on typos
continous = "continous"

# names and non-english words
falso = "falso"
yoh = "yoh"
tak = "tak"
lacker = "lacker"
fonctions = "fonctions"
calle = "calle"
gool = "gool"
sur = "sur"
periodes = "periodes"
waring = "waring"
espaces = "espaces"
yau = "yau"
topologie = "topologie"
claus = "claus"
parth = "parth"
ines = "ines"
fike = "fike"
tunnell = "tunnell"
groupes = "groupes"
htmp = "htmp"
ket = "ket"
clos = "clos"
abd = "abd"

# typo that i'm worried about breaking
conciousness = "conciousness" # part of a test
assum = "assum" # this might just be an abbrv but it's in a tactic and not used anywhere else

[files]
extend-exclude = ["docs/references.bib"]
```

I've dropped this config from this PR, but these typos ran across the repository - it may be beneficial for there to be CI to stop these unnecessary theorem deprecations. However, as the config also indicates, this captures a lot of false positives from the various abbreviations used.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Covering/Vitali.lean [hunk 1]
```diff
@@ -63,6 +63,10 @@ theorem exists_disjoint_subfamily_covering_enlargment (B : ι → Set α) (t : S
   the already chosen one, pick another element with large `δ`. Go on forever (transfinitely) until
   there is nothing left.
 
+@[deprecated (since := "2024-12-25")]
```
Comment (grunweg) [inline Mathlib/MeasureTheory/Covering/Vitali.lean:66]: These lines are in the middle of the proof: can you please move them to a proper place? Thanks!

File: Mathlib/MeasureTheory/Covering/Vitali.lean [hunk 2]
```diff
@@ -153,7 +157,7 @@ theorem exists_disjoint_subfamily_covering_enlargment (B : ι → Set α) (t : S
 /-- Vitali covering theorem, closed balls version: given a family `t` of closed balls, one can
 extract a disjoint subfamily `u ⊆ t` so that all balls in `t` are covered by the τ-times
 dilations of balls in `u`, for some `τ > 3`. -/
-theorem exists_disjoint_subfamily_covering_enlargment_closedBall
+theorem exists_disjoint_subfamily_covering_enlargement_closedBall
```
Comment (grunweg) [inline Mathlib/MeasureTheory/Covering/Vitali.lean:160]: Please add a deprecation warning for this result.

File: Mathlib/MeasureTheory/Integral/SetIntegral.lean [hunk 3]
```diff
@@ -1492,7 +1492,7 @@ theorem measure_le_lintegral_thickenedIndicator (μ : Measure X) {E : Set X}
 
 end thickenedIndicator
 
--- We declare a new `{X : Type*}` to discard the instance `[MeasureableSpace X]`
+-- We declare a new `{X : Type*}` to discard the instance `[measurableSpace X]`
```
Comment (grunweg) [inline Mathlib/MeasureTheory/Integral/SetIntegral.lean:1495]: Please revert this change (or explain why you think it's good).

**Review by grunweg — COMMENTED** [review]
Thanks! Looks good, with three small comments addressed.
maintainer delegate

**Comment by grunweg** [comment]
Also, some automated checks are failing as you have some lines longer than 100 characters. Can you fix those warnings as well, please?

File: Archive/Wiedijk100Theorems/BallotProblem.lean [hunk 4]
```diff
@@ -165,9 +165,11 @@ theorem disjoint_bits (p q : ℕ) :
 
 open MeasureTheory.Measure
 
-private def measureableSpace_list_int : MeasurableSpace (List ℤ) := ⊤
+private def measurableSpace_list_int : MeasurableSpace (List ℤ) := ⊤
```
Comment (urkud) [inline Archive/Wiedijk100Theorems/BallotProblem.lean:168]: I think that `private def`s can go away without deprecations.

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge
